appmap/test/normalize.py

```diff
@@ -46,6 +46,13 @@ def normalize_metadata(metadata):
                 v = f.pop('version')
                 assert v == pytest.__version__
 
+def normalize_headers(dct):
+    """Remove some headers which are variable between implementations.
+    This allows sharing tests between web frameworks, for example.
+    """
+    for hdr in ['User-Agent', 'Content-Length', 'ETag', 'Cookie', 'Host']:
+        dct.pop(hdr, None)
```

Comment:
I think it's worth confirming that the values of these keys are of the correct type, as we're doing for `elapsed` and `object_id` below